### PR TITLE
Removed django-registration-redux

### LINKED ISSUE #104: Remove django-registration-redux
**FEATURE REQUEST**:

**Description**:
To keep things clean and in order, removal of django-registration-redux before creating a custom-registration model.

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
**Status**:
- [x] Ready
- [ ] In Development
- [ ] On Hold

**Which issue does this PR fix?**: 
fixes #104 

Brief description of what this PR does:
This PR  is for removing django-registration-redux model.

### CODE REVIEW

File: oshc/oshc/settings.py [hunk 1]
```diff
@@ -27,9 +26,9 @@
 
 # Tuple of people who get error notifications
 ADMINS = [
-    ('Tapasweni Pathak','tapaswenipathak@gmail.com'),
-    ('Nikhita Raghunath','nikitaraghunath@gmail.com'),
-    ('Ibrahim Jarif','jarifibrahim@gmail.com'),
+    ('Tapasweni Pathak', 'tapaswenipathak@gmail.com'),
+    ('Nikhita Raghunath', 'nikitaraghunath@gmail.com'),
+    ('Ibrahim Jarif', 'jarifibrahim@gmail.com'),
```
Comment (anubhakushwaha) [inline oshc/oshc/settings.py:31]: @vaibhavsingh97  Please remove these spaces

Comment (vaibhavsingh97) [reply oshc/oshc/settings.py:31]: @anubhakushwaha According to PEP 8, we should have spaces. If you want i can remove.

Comment (anubhakushwaha) [reply oshc/oshc/settings.py:31]: @vaibhavsingh97 Ooh that's interesting, we would love to learn more,Is this the place? https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

File: oshc/oshc/settings.py [hunk 2]
```diff
@@ -91,24 +84,6 @@
...
-# If True, the user will be automatically logged in.
-REGISTRATION_AUTO_LOGIN = True
-# If true, email will be sent as HTML
-# REGISTRATION_EMAIL_HTML = True
-# The page you want users to arrive at after they successfully log in
-# The page users are directed to if they are not logged in,
-LOGIN_REDIRECT_URL = 'home'
-# and are trying to access pages requiring authentication
-LOGIN_URL = '/accounts/login/'
-LOGOUT_REDIRECT_URL = 'home'
```
Comment (anubhakushwaha) [inline oshc/oshc/settings.py]: @vaibhavsingh97  Remove this extra line

**Review by anubhakushwaha — APPROVED** [review]
Just the minor changes, else LGTM

**Comment by anubhakushwaha** [comment]
@vaibhavsingh97  Further remove the unit tests

**Review by anubhakushwaha — CHANGES_REQUESTED** [review]
Remove unit tests for now

**Comment by vaibhavsingh97** [comment]
@anubhakushwaha I had made suggested change. Please merge

File: oshc/oshc/settings.py (hunk 1 above)
Comment (vaibhavsingh97) [reply oshc/oshc/settings.py:31]: @anubhakushwaha Refer here: https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

**Review by anubhakushwaha — APPROVED** [review]

**Comment by anubhakushwaha** [comment]
@vaibhavsingh97 Just mention the reason for the decrease in coverage for everyone and it's ready to be merged.

**Comment by vaibhavsingh97** [comment]
@anubhakushwaha @tapasweni-pathak @nikhita Please find the detailed report 📓 
### Reason for decrease in coverage
#### Before removal of test case:

| Name                            | Stmts | Miss | Cover |
|---------------------------------|-------|------|-------|
| main/__init__.py                | 0     | 0    | 100%  |
| main/admin.py                   | 6     | 0    | 100%  |
| main/apps.py                    | 4     | 4    | 0%    |
| main/migrations/0001_initial.py | 6     | 0    | 100%  |
| main/migrations/__init__.py     | 0     | 0    | 100%  |
| main/models.py                  | 12    | 1    | 92%   |
| main/regbackend.py              | 17    | 0    | 100%  |
| main/tests.py                   | 46    | 0    | 100%  |
| main/urls.py                    | 3     | 0    | 100%  |
| main/views.py                   | 8     | 0    | 100%  |
| Total                           | 102   | 5    | 95%   |

Statement covered: 102
Missed Statement: 5
Percentage: (102-5)/102 * 100 = **`95%`**

#### After removal of test case:

| Name                            | Stmts | Miss | Cover |
|---------------------------------|-------|------|-------|
| main/__init__.py                | 0     | 0    | 100%  |
| main/admin.py                   | 6     | 0    | 100%  |
| main/apps.py                    | 4     | 4    | 0%    |
| main/migrations/0001_initial.py | 6     | 0    | 100%  |
| main/migrations/__init__.py     | 0     | 0    | 100%  |
| main/models.py                  | 12    | 1    | 92%   |
| main/tests.py                   | 12    | 0    | 100%  |
| main/urls.py                    | 3     | 0    | 100%  |
| main/views.py                   | 8     | 0    | 100%  |
| Total                           | 51    | 5    | 90%   |


Statement covered: 51
Missed Statement: 5
Percentage: (51-5)/51 * 100 = **`90%`**

**Comment by anubhakushwaha** [comment]
Merging, keeping the two commits separate just as a reminder that we have removed the test cases as well.

**Pull request merged**